kegg-pathways-completeness-tool/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.14
+
+LABEL Maintainer="Ekaterina Sakharova <kates@ebi.ac.uk>"
+
+# Install dependencies and Git
+RUN apk add --no-cache --update-cache --update python3 gzip git && \
+    apk add --no-cache --update py3-pip && \
+    apk add --no-cache --update bash && \
```

Comment:
We need to install the nextflow required packages, I don't remember the list of them for alpine. I'll get them later

Reply:
true, I know what they are, will add